backend/submissions/views.py

```
@@ -0,0 +1,399 @@
+                if not pdf_file:
+                    raise ValueError("No PDF file provided")
+
+                # Read PDF content into memory
+                pdf_content = pdf_file.read()
+
+                # Upload PDF to Supabase Storage first
+                supabase = SupabaseSubmission()
+                bucket_path = f"pdfs/{pdf_file.name}"
+                
+                # Upload to Supabase Storage using the content
+                storage_response = supabase.supabase.storage.from_("data_files").upload(
```

Ruff (0.11.9)

Local variable `storage_response` is assigned to but never used

Remove assignment to unused variable `storage_response`

(F841)